sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -197,6 +197,15 @@ static const struct snd_soc_acpi_link_adr tgl_rvp[] = {
 	{}
 };
 
+static const struct snd_soc_acpi_link_adr tgl_hp_rvp[] = {
```

Comment:
that's pretty funny.

You copied the line below which was intended to mean "TGL for Hewlett-Packard" (HP).

This should be modified to tgl_h_rvp ;-)

But let's step back: why doesn't the RVP has the RT1301 amplifiers? Is it a valid configuration to have the headset only? If yes, maybe we should just have  "tgl_rvp_headset_only"

Reply:
@plbossart  Sorry, this is my mistake, ”tgl_hp_rvp“ is a wrong name!

 TGL-H-RVP doesn‘t have RT1308. When there is only the headset, it is valid on TGL-H-RVP, and I have verified it.
Do I need to change it to "tgl_rvp_headset_only"？

I work for Intel, and I will explain to @mengdonglin why I need to submit this PR!

Reply:
> Do I need to change it to "tgl_rvp_headset_only"？

yes please.